src/client/a2/main/grid/layers/filters/radar-filter.ts

```diff
@@ -0,0 +1,88 @@
+import * as PIXI from 'pixi.js'
+
+export class RadarFilter extends PIXI.Filter {
+  constructor(private maxTime: number) {
+    super(
```

Comment:
True. Technically these can be managed as "assets". Currently, we don't have a system to do this.